OpenSim/Common/ComponentPath.cpp

```diff
@@ -28,7 +28,7 @@ using namespace std;
 
 // Set static member variables.
 const char ComponentPath::separator = '/';
-const std::string ComponentPath::invalidChars = "\\/*+";
+const std::string ComponentPath::invalidChars = "\\/*+ \t\n";
```

Comment:
Is `:` still used anywhere for aliases?

Reply:
It is. I would like the name checker to also invalidate for `:,;` even though it is OK for an Output path to use `:` to denote the alias. I have not resolved precisely how to handle that yet.

Reply:
> I have not resolved precisely how to handle that yet.

Planned for this PR, a separate PR, noted in an Issue, not important, …?

Reply:
I created issue #2007 to punt 🏉 to a follow-up PR.

Reply:
It's good! Fine to tackle that later.